### PR TITLE
Parsers for the rest of the commands

### PR DESCRIPTION
**Pull request by lkldev**
This PR will contain parsers for the rest of the commands.

However, I have not tested it out, so please do not merge yet.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DisenrolCommand.java [hunk 1]
```diff
@@ -27,13 +27,13 @@
...
-            + "\n\t1. " + COMMAND_WORD + " "
+            + "\n\t\t1. " + COMMAND_WORD + " "
             + PREFIX_CLASS_INDEX + "1 "
             + PREFIX_STUDENT + "all "
-            + "\n2. " + COMMAND_WORD + " "
+            + "\n\t\t2. " + COMMAND_WORD + " "
             + PREFIX_CLASS_INDEX + "1 "
             + PREFIX_STUDENT + "1,2,3 "
-            + "\n3. " + COMMAND_WORD + " "
+            + "\n\t3. " + COMMAND_WORD + " "
```
Comment (jxt00) [inline src/main/java/seedu/address/logic/commands/DisenrolCommand.java:36]: Is a `\t` needed for this, like the rest?

File: src/main/java/seedu/address/logic/parser/DisenrolCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package seedu.address.logic.parser;
+
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.DisenrolCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.Model;
+
+/**
+ * Parses input arguments and creates a new EnrolCommand object
```
Comment (jxt00) [inline src/main/java/seedu/address/logic/parser/DisenrolCommandParser.java:9]: Should be `DisenrolCommand`.

File: src/main/java/seedu/address/logic/parser/DisenrolCommandParser.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+ */
+public class DisenrolCommandParser implements Parser<Command> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the EnrolCommand
+     * and returns a EnrolCommand object for execution.
```
Comment (jxt00) [inline src/main/java/seedu/address/logic/parser/DisenrolCommandParser.java:15]: Same for these.

Comment (lkldev) [reply src/main/java/seedu/address/logic/parser/DisenrolCommandParser.java]: Resolved in latest commit

File: src/main/java/seedu/address/logic/parser/DisenrolCommandParser.java (hunk 2 above)
Comment (lkldev) [reply src/main/java/seedu/address/logic/parser/DisenrolCommandParser.java]: Resolved in latest commit

File: src/main/java/seedu/address/logic/commands/DisenrolCommand.java (hunk 1 above)
Comment (lkldev) [reply src/main/java/seedu/address/logic/commands/DisenrolCommand.java]: Resolved in latest commit

**Review by wxliong — APPROVED** [review]
LGTM! :)

**Review by Gernene — APPROVED** [review]

**Pull request merged**